packages/react-components/react-persona/src/stories/Persona/PersonaTextAlignment.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Persona } from '@fluentui/react-persona';
+import { makeStyles } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(3, 1fr)',
+    justifyItems: 'center',
+  },
+});
+
+export const TextAlignment = () => {
+  const styles = useStyles();
+
+  return (
+    <div className={styles.root}>
+      <Persona
+        textAlignment="start"
```

Comment:
Is text alignment difficult to do without this prop? I'm afraid to start adding props that are just mimicking things we can already do with css.

Reply:
Because of this portion of code needed for the text alignment, it is not straightforward. The presence has to be aligned to the primary text when the size is <= large. To achieve this, I am changing the way the grid is working for these sizes. This is needed since the tokens wouldn't align with the top padding needed to align it correctly. If we were to use tokens, we would end up having to do something like this: `topPadding: calc(${tokens.verticalSpacingXYZ} + 1px)`

```tsx
  // These alignToPrimary styles are needed due to presence being too small to center with the primary text.
  alignToPrimary: {
    gridTemplateColumns: `max-content [middle] max-content`,
  },
  afterAlignToPrimary: {
    alignSelf: 'center',
    gridRowStart: 'unset',
    gridColumnEnd: 'middle',
  },
  beforeAlignToPrimary: {
    alignSelf: 'center',
    gridRowStart: 'unset',
    gridColumnStart: 'middle',
  },
  ```